tests/accessor/generic_accessor_api_common.h

```diff
@@ -270,7 +273,9 @@ class run_api_tests {
       auto offset_id =
           util::get_cts_object::id<dims>::get(offset, offset, offset);
       std::remove_const_t<T> data[buff_size];
-      std::iota(data, (data + buff_range.size()), 0);
+      for (size_t i = 0; i < buff_size; i++) {
```

Comment:
`std::copy`

Reply:
std::copy doesn't seem to work for this purpose, changed to std::generate